spec/services/discovery_engine/quality/sample_query_set_spec.rb

```diff
@@ -7,7 +7,7 @@
     allow(DiscoveryEngine::Clients).to receive_messages(sample_query_set_service: sample_query_set_service_stub, sample_query_service: sample_query_service_stub)
   end
 
-  let(:sample_query_set_service_stub) { double("sample_query_set_service", create_sample_query_set: nil) }
+  let(:sample_query_set_service_stub) { double("sample_query_set_service", create_sample_query_set: nil, get_sample_query_set: nil) }
```

Comment:
We will need to update the stubbed response to the `get_sample_query_set` message. It will pass if we get it to return true rather than nil, but I think it's probably nicer and more informative to give a version of what the real response is.

eg

```
let(:response_object) do
  {
    "name": "projects/123/locations/global/sampleQuerySets/clickstream 2025-10",
    "displayName": "clickstream 2025-10",
    "createTime": "a-time-stamp"
  }
end
```